draft-ietf-tls-tls13.md

```diff
@@ -2124,24 +2127,31 @@ mode, the server MUST send Certificate ({{certificate}}) and CertificateVerify
 {:/comment}
 {:br}
 
-In order to accept PSK key establishment, the server sends a
-"pre_shared_key" extension with the selected identity.
+Each PSK MUST be associated with a single Hash algorithm. For PSKs established
+via the ticket mechanism ({{NewSessionTicket}}), this is the Hash used for
+the KDF. For externally established PSKs, the Hash algorithm is set when the
+PSK is established.
+
+Prior to accepting PSK key establishment, the server MUST validate the
+"hello_finished" extension. If this extension is not present, the server
+MUST send a fatal "missing_extension" alert. If it does not validate, the
+server MUST send a fatal "decrypt_error" alert.
+
+In order to accept PSK key establishment, the server sends an empty
```

Comment:
Remind me why there is no value from the server sending a MAC as well for key confirmation?